virt/getting_started/virt-using-the-cli-tools.adoc

```diff
@@ -28,3 +28,7 @@ include::modules/virt-deploying-libguestfs-with-virtctl.adoc[leveloffset=+1]
 
 include::modules/virt-about-libguestfs-tools-virtctl-guestfs.adoc[leveloffset=+2]
 
+[id="using_ansible__{context}"]
+== Using Ansible
+To use the Ansible collection for {VirtProductName}, see link:https://console.redhat.com/ansible/automation-hub/repo/published/redhat/openshift_virtualization[Red Hat Ansible Automation Hub] ({hybrid-console}).
```

Comment:
Nit - per the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces), we should have a non-breaking space in between Red and Hat:
```suggestion
To use the Ansible collection for {VirtProductName}, see link:https://console.redhat.com/ansible/automation-hub/repo/published/redhat/openshift_virtualization[Red{nbsp}Hat Ansible Automation Hub] ({hybrid-console}).
```